src/storage-controller/src/lib.rs

```diff
@@ -948,6 +963,7 @@ where
                 data_source,
                 collection_metadata: metadata,
                 extra_state,
+                active_copies: 1,
```

Comment:
How do we know that there is one active copy when a collection is created? Don't we need to look at the number of replicas for that?

Reply:
I see. Would be great to have that context in a comment too!